### PR TITLE
SAK-47291 Conversion scripts for avoiding i18n issues when using external tools in gradebook

### PR DESCRIPTION
**Pull request by josecebe**
This is linked to https://github.com/sakaiproject/sakai/pull/10540

### CODE REVIEW

File: docs/conversion/sakai_21_3-21_4_mysql_conversion.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-- SAK-47291
```
Comment (josecebe) [inline docs/conversion/sakai_21_3-21_4_mysql_conversion.sql:1]: Is this the correct file to put this conversion script? I think my fix should be applied to 21.x, so next version of 21 will be 21.4

File: docs/conversion/sakai_21_3-21_4_oracle_conversion.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+-- SAK-47291
+UPDATE gb_gradable_object_t SET EXTERNAL_APP_NAME = 'sakai.assignment.grades' WHERE EXTERNAL_ID LIKE '/assignment/a/%';
+UPDATE gb_gradable_object_t SET EXTERNAL_APP_NAME = 'sakai.lessonbuildertool' WHERE EXTERNAL_ID LIKE 'lesson-builder:%';
+UPDATE gb_gradable_object_t SET EXTERNAL_APP_NAME = 'sakai.attendance' WHERE EXTERNAL_ID LIKE 'sakai.attendance.%';
+UPDATE gb_gradable_object_t SET EXTERNAL_APP_NAME = 'sakai.samigo' WHERE REGEXP_LIKE(EXTERNAL_ID, '^[0-9]+$');
```
Comment (josecebe) [inline docs/conversion/sakai_21_3-21_4_oracle_conversion.sql:5]: I'm not used to use oracle, do you think whis query is valid? The intention is to find all the rows containing numbers. These rows will be the ones that are refering to a samigo gradebook item.

Comment (josecebe) [reply docs/conversion/sakai_21_3-21_4_oracle_conversion.sql:5]: Thanks @bjones86

**Comment by mpellicer** [comment]
I think this must be in 21.3 - 21.4 and 22.0 - 22.1, not many institutions have 22 yet but if they upgrade from 22.0 to 22.1 they will miss this conversion.

**Comment by josecebe** [comment]
@mpellicer I have added the conversion script for 22.0 to 22.1. Thanks!

**Review by mpellicer — APPROVED** [review]

**Comment by plukasew** [comment]
@josecebe @mpellicer it looks like this has gone missing from the 22 conversion scripts. It is still needed, correct?

**Comment by mpellicer** [comment]
@plukasew If it's present in 21.4 to 21.5 it's going to be applied before 22 right?

It was removed here, I don't know the reason, @ern?

https://github.com/sakaiproject/sakai-reference/commit/8f3b0eea2b12c488eb7caa8867a7291bc08e72ce

**Comment by plukasew** [comment]
Actually no, 21.5 has not been released so I would normally not apply it. I only looked at the script out of curiosity and expected to find it in the 22 scripts.

**Pull request merged**